lib/internal/process/esm_loader.js

```diff
@@ -40,8 +45,14 @@ async function importModuleDynamicallyCallback(wrap, specifier, assertions) {
   throw new ERR_VM_DYNAMIC_IMPORT_CALLBACK_MISSING();
 };
 
-const esmLoader = new ESMLoader();
-exports.esmLoader = esmLoader;
+debug('hooking up public ESMLoader');
+const customLoaders = getOptionValue('--experimental-loader');
```

Comment:
Please avoid accessing `getOptionValue()` at the top level in a module that's supposed to be `require()`ed. That makes the module immediately runtime-dependent and unsnapshottable. The loader can be created when the loader initialization is actually run for the first time, see https://github.com/nodejs/node/pull/45828.